encryption/aes/aes.go

```diff
@@ -0,0 +1,86 @@
+package aes
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"io"
+)
+
+const KeyLength = 32
+
+// Encrypt encrypts data using 256-bit AES-GCM, key must have length 32 or more
+func Encrypt(plaintext string, key string) (ciphertext string, err error) {
```

Comment:
@viettd maybe we can make this function 
```
func Encrypt(data []byte, key string) (encrypted []byte, err error)
```
And Add a new one
```
func EncryptString(plaintext string, key string) (ciphertext string, err error) {
```

same to the `Decrypt` part

Reply:
Added